frontend/src/views/NftDisplay.vue

```diff
@@ -46,22 +46,147 @@
             :showFavoriteWeaponsDefVal="false"
             :canFavorite="false"
             :newWeapon="true"
-          />
+          >
+            <template #above="{ weapon: { id } }">
+              <div
+                class="
+                  d-flex
+                  flex-column
+                  align-items-center
+                  justify-content-center
+                  m-top-negative-5
+                "
+              >
+                <span class="d-block text-center fix-h24" v-if="nftPriceInSkill() !== '0'">
+                  <span
+                    v-if="nftPriceInSkill() !== '0'"
+                    v-tooltip.top="{
+                      content: maxPrecisionSkill(nftPrice),
+                      trigger: isMobile() ? 'click' : 'hover',
+                    }"
+                    @mouseover="hover = !isMobile() || true"
+                    @mouseleave="hover = !isMobile()"
+                  >
+                    <strong>Price</strong>:
+                    <CurrencyConverter :skill="nftPriceInSkill()" />
+                  </span>
+                </span>
+                <b-button
+                  v-if="nftPriceInSkill() !== '0'"
+                  @click="
+                    nftId = id;
+                    purchaseNft();
+                  "
+                  variant="primary"
+                  class="gtag-link-others"
+                >
+                  {{ nftPriceInSkill() !== "0" ? "Purchase" : "Sold" }}
```

Comment:
The condition is already in v-if of this b-button. Condition here won't have any effect - does it break any functionality (Puchase/Sold text)?